### PR TITLE
up: validate usage of enable on initialized cluster

### PR DESCRIPTION
**Pull request by mfojtik**
When the cluster is already initialized in baseDir, we should not allow to specify `--enable` flag but users should use oc cluster add instead.

/cc @deads2k

### CODE REVIEW

File: pkg/oc/bootstrap/docker/up.go [hunk 1]
```diff
@@ -390,6 +390,13 @@ func (c *ClusterUpConfig) Validate(errout io.Writer) error {
 	if c.dockerClient == nil {
 		return fmt.Errorf("missing dockerClient")
 	}
+	if c.command.Flag("enable").Changed {
```
Comment (deads2k) [inline pkg/oc/bootstrap/docker/up.go]: Naughty!  We shouldn't keep that command after `Complete`.  Just check if the value of `UserEnabledComponents` has changed from the default we choose.

Comment (mfojtik) [reply pkg/oc/bootstrap/docker/up.go]: @deads2k well this is Validate() which is before Complete. What is wrong on relying on flag library telling us whether a flag was changed or not?

Comment (mfojtik) [reply pkg/oc/bootstrap/docker/up.go]: OK, I was wrong... I always thought we do Validate() before we complete, in cluster up it is other way around...

**Pull request merged**